README.md

```diff
@@ -13,3 +13,13 @@ The output offence code data is generated by combining a number of different dat
 ## Consistently Formatting Data
 In order to make differences between versions of data easy to read, the data should be sorted before committing. This can be done by running `./data-formatter/format.sh` from the root of the repository. This will sort the arrays of data by alphabetical attribute name and then output them with their attributes sorted.
 
+## Publishing
+The output-data lib is published as an NPM package to [GitHub packages](https://github.com/features/packages) using a [GitHub Actions](https://github.com/features/actions) workflow `.github/workflows/release.yml`. 
+
+Versioning and publishing is automated using [semantic-release](https://github.com/semantic-release/semantic-release). The following commit convention convention triggers releases:
```

Comment:
So are we saying that the commit message has to match a certain format in order for the npm package to be released?

Reply:
Yes, more [info](https://github.com/semantic-release/semantic-release#Commit-message-format)

Reply:
What happens if the commit message doesn't match the format? Do we release with a default release type? Or does nothing get released?

Reply:
Nothing gets released